lib/config/routes/router.dart

```diff
@@ -19,7 +20,7 @@ import 'navigator_helper.dart';
 abstract class AppRouter {
   static Map<String, Widget Function(BuildContext)> mapRoutes() {
     return {
-      AppRoutes.initial: (_) => const TemplatePage(),
+      AppRoutes.helpCenter: (_) => const HelpCenter(),
```

Comment:
Why there are routes `helpCenter` and `help` even though they're navigating to the same place. Remove one of them.